packages/examples/packages/interactive-ui/src/components/InteractiveForm.tsx

```diff
@@ -46,54 +48,72 @@ export type InteractiveFormState = {
   'example-selector': string;
 };
 
-export const InteractiveForm: SnapComponent = () => {
+export const InteractiveForm: SnapComponent<{ disabled?: boolean }> = ({
+  disabled,
```

Comment:
Maybe just add a few disabled elements instead of a prop to disable everything?

Reply:
Ideally we want to show each component's disabled state, it would duplicate a lot to introduce each component in its disabled state too 🤔

Reply:
IMO it is neat that we can test the submit button being disabled too, would be kinda hard without a disabled state